geo/Cargo.toml

```diff
@@ -24,7 +24,10 @@ voronoi = ["delaunay"]
 earcutr = { version = "0.4.2", optional = true }
 spade = { version = "2.2.0", optional = true }
 float_next_after = "1.0.0"
-geo-types = { version = "0.7.9", features = ["approx", "use-rstar_0_11"] }
+geo-types = { version = "0.7.11", features = [
```

Comment:
Shouldnt we have published geo-types to cloudsmith as something like `0.7.12-alpha1` ?

Reply:
This change just bumps geo-types to the latest version

Reply:
This has been addressed by rebasing against upstream